logic/concludable/ConjunctionConcludable.java

```diff
@@ -50,36 +52,37 @@ private ConjunctionConcludable(CONSTRAINT constraint) {
         return new Extractor(conjunction.variables()).concludables();
     }
 
-    public Stream<Pair<Rule, Unification>> findUnifiableRules(Stream<Rule> allRules) {
+    public Stream<Pair<Rule, Map<Reference, Set<Reference>>>> findUnifiableRules(Stream<Rule> allRules) {
```

Comment:
The unification object will soon just take a mapping at its construction. So for now we'll just return the mappings.

Reply:
don't need to call it `find`, it's just `unifiableRules()`